### DISCUSSION TITLE
The link for the android download leads to a 404, is this intentional?

### DISCUSSION THREAD

**Discussion by totallywaffles** [ppy/osu#26864] [Q&A] [answered]
Title

**Comment by giyugg** [comment, marked as answer]
[bdach](https://github.com/bdach) commented [20 hours ago](https://github.com/ppy/osu/issues/26800#issuecomment-1916749640) - [reference](https://github.com/ppy/osu/issues/26800)
The mobile builds aren't there just yet, but they're building, please stay tuned.
